### ISSUE TITLE
Integration tests are broken

### ISSUE THREAD

**Issue by budhrg** [minishift/minishift#205]
It looks like integration tests are broken.

```
$ make integration
go test github.com/minishift/minishift/test/integration --tags=integration
# github.com/minishift/minishift/test/integration
test/integration/persistence_test.go:50: kubectlRunner.GetPod undefined (type *"github.com/minishift/minishift/test/integration/util".KubectlRunner has no field or method GetPod)
FAIL	github.com/minishift/minishift/test/integration [build failed]
Makefile:93: recipe for target 'integration' failed
make: *** [integration] Error 2
```

These seems to be taken as it is from [minkube](https://github.com/kubernetes/minikube/tree/master/test/integration)

Even removing the `persistence_test.go`, other tests are still broken:
```
make integration                                                           
go test github.com/minishift/minishift/test/integration --tags=integration
--- FAIL: TestAddons (0.00s)
	util.go:69: Error running command: status --format={{.MinikubeStatus}} fork/exec /home/budhram/gowork/src/github.com/minishift/minishift/out/minikube: no such file or directory. Output: 
--- FAIL: TestDashboard (0.00s)
	util.go:69: Error running command: start  fork/exec /home/budhram/gowork/src/github.com/minishift/minishift/out/minikube: no such file or directory. Output: 
--- FAIL: TestClusterDNS (0.00s)
	util.go:69: Error running command: status --format={{.MinikubeStatus}} fork/exec /home/budhram/gowork/src/github.com/minishift/minishift/out/minikube: no such file or directory. Output: 
--- FAIL: TestClusterEnv (0.00s)
	util.go:69: Error running command: status --format={{.MinikubeStatus}} fork/exec /home/budhram/gowork/src/github.com/minishift/minishift/out/minikube: no such file or directory. Output: 
--- FAIL: TestClusterLogs (0.00s)
	util.go:69: Error running command: status --format={{.MinikubeStatus}} fork/exec /home/budhram/gowork/src/github.com/minishift/minishift/out/minikube: no such file or directory. Output: 
--- FAIL: TestClusterSSH (0.00s)
	util.go:69: Error running command: status --format={{.MinikubeStatus}} fork/exec /home/budhram/gowork/src/github.com/minishift/minishift/out/minikube: no such file or directory. Output: 
--- FAIL: TestClusterStatus (0.00s)
	util.go:69: Error running command: status --format={{.MinikubeStatus}} fork/exec /home/budhram/gowork/src/github.com/minishift/minishift/out/minikube: no such file or directory. Output: 
--- FAIL: TestDockerEnv (0.00s)
	util.go:69: Error running command: delete fork/exec /home/budhram/gowork/src/github.com/minishift/minishift/out/minikube: no such file or directory. Output: 
--- FAIL: TestStartStop (0.00s)
	util.go:69: Error running command: status --format={{.MinikubeStatus}} fork/exec /home/budhram/gowork/src/github.com/minishift/minishift/out/minikube: no such file or directory. Output: 
FAIL
FAIL	github.com/minishift/minishift/test/integration	0.007s
Makefile:93: recipe for target 'integration' failed
make: *** [integration] Error 1
```

**Comment by budhrg** [comment]
Might be related to minikube binary which is obsolute and we need new set of tests.

**Comment by hferentschik** [comment]
This is a known issue. Integration tests are not working atm. See also https://github.com/minishift/minishift/issues/135. The tests need to be revised and adjusted to minishift. We will also need to find the right framework to use.

**Comment by hferentschik** [comment]
Merged via pull request #243. This pull request cut down the existing integration tests to a minimal working subset. Further refinement of the integration testwork via issue #135
